vignettes/contribution_model.Rmd

```diff
@@ -44,16 +42,14 @@ Type 1 Contribution with Code:
 
  * First, the user creates an issue or comments on an existing issue to notify that they’d like to contribute code.
  
- * User creates a new branch within the development branch (devel) and creates a [pull request](pr_review_guidance.html) for review [github usage](git_usage.html).
-
- * Code should follow our [programming strategy](programming_strategy.html) and [unit test guidance](unit_test_guidance.html).
+ * Follow our [development process](https://roche-gsk.github.io/admiral/articles/development_process.html) step-by-step guide.
```

Comment:
I would remove `https://roche-gsk.github.io/admiral/articles/`. Then it works locally.

Reply:
Removed